.eslintrc.js

```diff
@@ -20,5 +20,11 @@ module.exports = {
     ],
     '@typescript-eslint/no-namespace': 0,
     '@typescript-eslint/no-var-requires': 0,
+    '@typescript-eslint/no-unused-vars': [
+      1,
+      {
+        argsIgnorePattern: '^_',
```

Comment:
That's a great suggestion!
What do you think if we scope this rule only to arguments? 

It's still possible to declare a variable starting with an underscore and forget to use it. In that case, we should get warnings from eslint. I think the current use-case is that we wish to handle arguments with this rule.

Reply:
To the best of my knowledge, this [`argsIgnorePattern`](https://eslint.org/docs/rules/no-unused-vars#argsignorepattern) option should only apply to arguments. There is another option called [`varsIgnorePattern `](https://eslint.org/docs/rules/no-unused-vars#varsignorepattern).

Reply:
You're completely right! My bad, I've confused the option name 👍 This looks great.